src/Evolution/Systems/ScalarWave/Equations.cpp

```diff
@@ -163,6 +169,14 @@ void UpwindFlux<Dim>::package_data(
     normal_times_flux_pi.get(d) =
         interface_unit_normal.get(d) * get(normal_dot_flux_pi);
   }
+  // Extract memory from package data, and populate it with quantities needed
```

Comment:
Because "extract memory" kind of sounds like other gravitational-wave things, I would suggest phrasing this as "Get references to storage in package_data, ..."

Reply:
rephrased more clearly!